app/controllers/concerns/recovery_code_concern.rb

```diff
@@ -0,0 +1,12 @@
+module RecoveryCodeConcern
+  delegate :active_profile, to: :current_user
+
+  def create_new_code
+    if active_profile.present?
+      Pii::ReEncryptor.new(current_user, user_session).perform
+      active_profile.recovery_code
```

Comment:
I'm saying if `Pii::ReEncryptor` is changed to return a recovery code (as opposed to `true` or `false`), it should probably be renamed to match its return value.